src/components/CreateGoalForm/CreateGoalform.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Text,
+  View,
+  StyleSheet,
+  TextInput,
+  Switch,
+  TouchableOpacity,
+} from 'react-native';
+import React, { useState } from 'react';
+import Deadline from './DatePicker';
+import Dropdown from './Dropdown';
+
+export default function CreateGoalform() {
+  const [title, onChangeTitle] = React.useState('');
+  const [description, onChangeDescription] = React.useState('');
```

Comment:
why are you using React.useState here, you have already imported it from react above so you can directly use useState

Reply:
Making the required changes.